gammapy/irf/edisp/core.py

```diff
@@ -48,9 +48,17 @@ class EnergyDispersion2D(IRF):
 
     tag = "edisp_2d"
     required_axes = ["energy_true", "migra", "offset"]
+    fill_value = 0.0
+
+    def _mask_out_bounds(self, invalid):
```

Comment:
Again why is this necessary? Doesn't it work to just declare `fill_value=0` for this case?

Reply:
without this addition `TestEDispKernel.test_to_image` and `test_edispkernel_from_1d` fail.  I think the extrapolation is needed if you are in the valid range of at least one of the energy axes.